docs/sources/administration/planning-iam-strategy/index.md

```diff
@@ -0,0 +1,213 @@
+### Service accounts
+
+You can use a service account to run automated workloads in Grafana, such as dashboard provisioning, configuration, or report generation. Create service accounts and service accounts tokens to authenticate applications, such as Terraform, with the Grafana API.
+
+> **Note:** Service accounts will eventually replace [API keys]({{< relref ".#api-keys" >}}) as the primary way to authenticate applications that interact with Grafana.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up an external SAML authentication provider
+- Interact with Grafana without signing in as a user
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+{{% admonition type="note" %}}
+Service accounts can only act in the organization they are created for. We recommend creating service accounts in each organization if you have the same task needed for multiple organizations.
+{{% /admonition %}}
+
+{{< vimeo 742056367 >}}
+<br>
+
+#### Service account tokens
+
+To authenticate with Grafana's HTTP API, a randomly generated string known as a service account token can be used as an alternative to a password.
+
+When a service account is created, it can be linked to multiple access tokens. These service access tokens can be utilized in the same manner as API keys, providing a means to programmatically access Grafana HTTP API.
+
+You can create multiple tokens for the same service account. You might want to do this if:
+
+- Multiple applications use the same permissions, but you want to audit or manage their actions separately.
+- You need to rotate or replace a compromised token.
+
+{{% admonition type="note" %}}
+In Grafana's audit logs it will still show up as the same service account.
+{{% /admonition %}}
+
+Service account access tokens inherit permissions from the service account.
+
+### API keys
+
+> **Note:** If you use Grafana v8.5 or newer, you should use service accounts instead of API keys. API keys will be deprecated in the near future. For more information, refer to [Grafana service accounts]({{< relref ".#service-accounts" >}}).
+
+You can use Grafana API keys to interact with data sources via HTTP APIs.
+
+## How to work with roles?
+
+Grafana utilizes roles to control the access of users and service accounts to specific resources and determine their authorized actions.
+
+You can assign roles through the user interface or APIs, establish them through Terraform, or synchronize them automatically via an external IAM provider.
+
+### What are roles?
```

Comment:
Don't we have role content that you can link to? What you have written is fine, just wondering about adding a link if folks want to explore more?